files/conf/k8s-extract

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -euo pipefail
+
+container=$1
+
+docker cp "$container":/kubelet /opt/bin/kubelet
```

Comment:
Changed from `docker cp "$container":/usr/local/bin/kubelet /opt/bin/kubelet` corresponding to https://github.com/giantswarm/hyperkube/blob/master/Dockerfile#L14